tests/NClient/NClient.Tests/ClientTests/TimeoutClientTest.cs

```diff
@@ -46,11 +47,11 @@ public async Task TimeoutClient_GetAsyncWithHttpClientTimeout_ThrowTaskCanceledE
 
             await nclient.Invoking(x => x.GetAsync(id))
                 .Should()
-                .ThrowExactlyAsync<TaskCanceledException>();
+                .ThrowExactlyAsync<OperationCanceledException>();
```

Comment:
Why did you change the exception type? Now the tests don't pass: 
> Expected type to be System.OperationCanceledException, but found System.Threading.Tasks.TaskCanceledException.

Reply:
There are some complex questions...